modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -563,6 +619,87 @@ test(
 	}
 );
 
+test(
+	'Create the page name with special characters makes the correct name appear in the path part of AC pages',
+	{
+		tag: '@LRAC-8988',
+	},
+
+	async ({apiHelpers, page}) => {
+		const pageTitle = 'Snúið Vinsælar þú';
+		const sitePage = await createSitePage({
+			apiHelpers,
+			pageTitle,
+		});
+
+		const channelName = 'My Property - ' + getRandomString();
+		await test.step('Connect the DXP to AC', async () => {
+			await syncAnalyticsCloud({
+				apiHelpers,
+				channelName,
+				page,
+			});
+		});
+
+		await test.step('Go to Snúið Vinsælar þú Page', async () => {
+			await navigateToSitePage({
+				page,
+				pageName: pageTitle,
+			});
+			await page.waitForTimeout(10000);
+		});
+
+		await test.step('Go to Analytics Cloud and Switch the property', async () => {
+			await navigateToACWorkspace({page});
+			await switchChannel({
+				channelName,
+				page,
+			});
+		});
+
+		await test.step('Go to Pages Tab', async () => {
+			await navigateTo({
+				page,
+				pageName: 'Pages',
+			});
+		});
+
+		await test.step('Change the time filter to Last 24 hours', async () => {
+			await changeTimeFilter({
+				page,
+				timeFilterPeriod: 'Last 24 hours',
+			});
+		});
+
+		await test.step('Access one of the pages on the list > Go to Path Tab', async () => {
+			await navigateTo({
+				page,
+				pageName: pageTitle,
+			});
+			await navigateTo({
+				page,
+				pageName: 'Path',
+			});
+		});
+
+		await test.step('Check that Snúið Vinsælar þú Page appear as referral page', async () => {
+			await expect(
+				page.getByText('Snúið Vinsælar ...', {exact: true}).first()
+			).toBeVisible({
+				timeout: 100 * 1000,
```

Comment:
@rafaellajordao , can you remove this timeout? if I'm no wrong, the default timeout for `toBeVisible` is 15s. I think it is enough time :smile: 
Also, if you look for that in the code, you'll see the other teams are avoiding to use it in this way

Reply:
of course! fixed! :smile: